clustered/server/src/main/java/org/ehcache/clustered/server/management/AbstractExposedStatistics.java

```diff
@@ -55,11 +52,19 @@
 @FindbugsSuppressWarnings("EQ_DOESNT_OVERRIDE_EQUALS")
 class AbstractExposedStatistics<T extends AliasBinding> extends AliasBindingManagementProvider.ExposedAliasBinding<T> implements Closeable {
 
+  private static final Map<String, SampleType> COMPOUND_SUFFIXES = new HashMap<String, SampleType>() {{
+    put("Count", SampleType.COUNTER);
+    put("Rate", SampleType.RATE);
+    put("LatencyMinimum", SampleType.LATENCY_MIN);
+    put("LatencyMaximum", SampleType.LATENCY_MAX);
+    put("LatencyAverage", SampleType.LATENCY_AVG);
+  }};
```

Comment:
I wouldn't use the `{{ ... }}` notation in production code as it creates a subclass of `HashMap`. Please use a `static { ... }` block instead.